src/theory/datatypes/datatypes_utils.h

```diff
@@ -0,0 +1,83 @@
+/******************************************************************************
```

Comment:
Note there is already `theory_datatypes_utils.h`.

Although this file should perhaps be called `tuple_utils.h`.

Reply:
File renamed to tuple_utils and class renamed to TupleUtils.